### ISSUE TITLE
https://zimfw.sh/install.zsh is not available

### ISSUE THREAD

**Issue by vitaly** [zimfw/zimfw.sh#3]
getting HTTP error 409

**Comment by ericbn** [comment]
I was looking into this and I'm afraid we currently cannot make this work with GitHub Pages. Only [client-side redirects](https://gist.github.com/domenic/1f286d415559b56d725bee51a62c24a7) are supported, but curl and wget won't process those. They will only understand server-side redirects.

Either we only use the full `https://raw.githubusercontent.com/zimfw/install/master/install.zsh` URL, or we use a different infrastructure that allows us to configure server-side redirects. S3, for example, [supports it](https://docs.aws.amazon.com/AmazonS3/latest/dev/how-to-page-redirect.html). Having a more flexible infrastructure for serving pages might also allow us to upload the [zsh-framework-benchmark results](https://github.com/zimfw/zsh-framework-benchmark/actions) to be publicly available, which is also something we had in mind.

@Eriner, thoughts?

**Comment by michaelhthomas** [comment]
Subdomaining might be another option, say install.zimfw.sh for example.

**Comment by michaelhthomas** [comment]
I'll go ahead and change the url for the time being

**Comment by ericbn** [comment]
Good point, @michaelhthomas! Redirecting on the subdomain level also works, and it's much simpler than the other options I mentioned!
